src/java.xml.crypto/share/classes/org/jcp/xml/dsig/internal/dom/Utils.java

```diff
@@ -67,10 +67,10 @@ private Utils() {}
      * @param i the Iterator
      * @return the Set of Nodes
      */
-    static Set<Node> toNodeSet(Iterator<Node> i) {
+    static Set<Node> toNodeSet(Iterator<?> i) {
```

Comment:
Why does `Iterator<Node>` cause a warning?

Reply:
It's called at https://github.com/wangweij/jdk/blob/604968474ac20abae1d65a5c00053c76a47c559a/src/java.xml.crypto/share/classes/org/jcp/xml/dsig/internal/dom/ApacheCanonicalizer.java#L159 where there is no template parameter.